### DISCUSSION TITLE
Directions on exposing devices to LXC Containers

### DISCUSSION THREAD

**Discussion by jeeftor** [tteck/Proxmox#695] [Ideas]
I installed the Zigbee2MQTT LXC container and followed the directions and realized I didn't have any devices present in the container. I plugged in a stick to the machine and could see it in my Proxmox console but not inside my LXC Container. 

Did some reading and realize I need to actually expose it to the container - does this belong in the directions? I'd be happy to PR add it but wanted to figure if its worth my time.

it also probably applies a few places so might make sense to anchor tag it somewhere

**Comment by tteck** [comment]
What are you proposing?
The Zigbee2MQTT LXC already does device passthrough in `zigbee2mqtt-v4.sh`
```
LXC_CONFIG=/etc/pve/lxc/${CTID}.conf
cat <<EOF >>$LXC_CONFIG
lxc.cgroup2.devices.allow: a
lxc.cap.drop: 
lxc.cgroup2.devices.allow: c 188:* rwm
lxc.cgroup2.devices.allow: c 189:* rwm
lxc.mount.entry: /dev/serial/by-id  dev/serial/by-id  none bind,optional,create=dir
lxc.mount.entry: /dev/ttyUSB0       dev/ttyUSB0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyUSB1       dev/ttyUSB1       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM0       dev/ttyACM0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM1       dev/ttyACM1       none bind,optional,create=file
EOF
```

**Reply by tteck** [reply]
Only works with privileged containers
